sdk/lib/atl/atlalloc.h

```diff
@@ -152,43 +152,43 @@ class CHeapPtr
 
     void Free()
     {
-        if (m_Data)
+        if (m_pData)
         {
-            Allocator::Free(m_Data);
-            m_Data = NULL;
+            Allocator::Free(m_pData);
+            m_pData = NULL;
         }
     }
 
     void Attach(T *lp)
     {
-        Allocator::Free(m_Data);
-        m_Data = lp;
+        Allocator::Free(m_pData);
+        m_pData = lp;
     }
 
     T *Detach()
     {
-        T *saveP = m_Data;
-        m_Data = NULL;
+        T *saveP = m_pData;
+        m_pData = NULL;
         return saveP;
     }
 
     T **operator &()
     {
-        ATLASSERT(m_Data == NULL);
-        return &m_Data;
+        ATLASSERT(m_pData == NULL);
+        return &m_pData;
     }
 
     operator T* () const
     {
-        return m_Data;
+        return m_pData;
     }
 
     T* operator->() const
     {
-        return m_Data;
+        return m_pData;
     }
 
-protected:
-    T *m_Data;
+public:
+    T *m_pData;
```

Comment:
This should be in CHeapPtrBase. https://docs.microsoft.com/en-us/cpp/atl/reference/cheapptrbase-class?view=vs-2019

Reply:
Yeah, but we don't have `CHeapPtrBase` yet,
and as long as there is no good reason to add it I want to keep this stuff as simple as possible :)